pkgs/tools/misc/less/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "less";
-  version = "563";
+  version = "581.2";
 
   src = fetchurl {
     url = "http://www.greenwoodsoftware.com/${pname}/${pname}-${version}.tar.gz";
-    sha256 = "16lsvk88vwjwp5ax1wnll44wxwnzs8lb2fn90xx2si64kwmnsnyf";
+    sha256 = "0fyqslvrasv19qjvqrwfwz2n7mnm93y61x9bcx09ga90mxyb8d6f";
   };
 
   configureFlags = [ "--sysconfdir=/etc" ] # Look for ‘sysless’ in /etc.
```

Comment:
Can we update the homepage to https://www.greenwoodsoftware.com/less/ ?

Reply:
Just to clarify here, it's a matter of switching http -> https

Reply:
> Can we update the homepage to [greenwoodsoftware.com/less](https://www.greenwoodsoftware.com/less/) ?

Sure, why not

> Just to clarify here, it's a matter of switching http -> https

Thanks for clarifying!